packages/vue-component/plugin/tag-handler.js

```diff
@@ -160,7 +160,7 @@ VueComponentTagHandler = class VueComponentTagHandler {
       }
 
       // Export
-      script = script.replace(jsExportDefaultReg, 'return');
+      script += `\nreturn require('${this.inputFile.getDisplayPath()}').default;`
```

Comment:
Ingenious!!!

On the other hand, this makes it synchronous? So no on demand loading? Or is this done at some other level?